modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -108,9 +130,14 @@ If you remove the `devworkspace-webhook-server` deployment without removing the
 +
 . Run the following commands to remove any lingering services, secrets, and config maps:
 +
+[NOTE]
+====
+Depending on the installation, some of the resources below may not be present on the cluster.
```

Comment:
- Extra whitespace before "some".
- "... some of the resources below ..." ---> "... some resources included in the following command might not be available on the cluster."
- Instead of a formal note, I would use an ordinary sentence.

Reply:
@sounix000 +1

Reply:
For the first point, by extra whitespace, do you mean it should be
> `Depending on the installation,  some of the resources`
?

Second point -- changed but I used `may not exist` instead of `not be available`. Available reads somewhat like "services may be unusable", etc.

Reply:
For the first point, I think he wants you to remove the extra space between "installation, some" if there is any.
'may not exist' sounds good, but I guess this change has not been made, as I see - 'might not be present on the cluster' in the PR.

Reply:
Hm, don't know how that got dropped -- sorry.